blog/loops.ftd

```diff
@@ -0,0 +1,92 @@
+-- import: fastn-community.github.io/bling/chat
+-- import: fastn.com/blog/authors
+-- import: fastn-community.github.io/doc-site/common
+-- import: fastn.com/utils
+-- import: fastn.com/content-library as lib
+
+-- common.post-meta index-meta: Understanding Loops
+published-on: November 8, 2023 at 3:29 pm
+post-url: /loops/
+author: $authors.rithik
+
+In this blog, we will be exploring on how `fastn` uses loops and
+what cool stuff you can do with them. Let's kick things
+off by using some lists. We'll explore and learn about
+different interesting aspects while working with them.
+
+- Looping using for loop syntax
+- Looping using $loop$ syntax
+
+-- common.post-meta meta: Understanding Loops
+published-on: November 8, 2023 at 3:29 pm
+post-url: /loops/
+author: $authors.rithik
+read-time: 5-10 mins
+
+In this blog, we will be exploring on how `fastn` uses loops and
+what cool stuff you can do with them. Let's kick things
+off by using some lists. We'll explore and learn about
+different interesting aspects while working with them.
```

Comment:
Can we combine `index-meta` and `meta` variables into one.

Reply:
I will do that